### PR TITLE
update tensorboard

### PR DESCRIPTION
**Pull request by eladc-git**
update tensor board visualization to get better graph visability

### CODE REVIEW

File: tests/pytorch_tests/function_tests/test_tensorboard_writer.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Sony Semiconductors Israel, Inc. All rights reserved.
+#
```
[the remark this answers is not in the document]
Comment (eladc-git) [reply tests/pytorch_tests/function_tests/test_tensorboard_writer.py:2]: Done

**Comment by eladc-git** [comment]
I see tests fail on OutOfMemory.

**Comment by reuvenperetz** [comment]
> I see tests fail on OutOfMemory.

Weird. Does it still happen when you comment out the newly added test from the tests-suite?

**Comment by eladc-git** [comment]
> 

I will try

**Comment by eladc-git** [comment]
So I remove the test from test_suite. It's Ok now.

**Pull request merged**